postrprocess_visual/visualizer.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import cv2
+
+PASCAL_CLASSES = ['aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable',
+                  'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']
+
```

Comment:
Can you get these classes from the dataset?

Reply:
You may leave these classes as a comment to clarify which classes are included in our dataset